vimrc

```diff
@@ -57,16 +67,11 @@ augroup vimrcEx
 
   " Allow stylesheets to autocomplete hyphenated words
   autocmd FileType css,scss,sass setlocal iskeyword+=-
-augroup END
 
-" Softtabs, 2 spaces
-set tabstop=2
-set shiftwidth=2
-set shiftround
-set expandtab
-
-" Display extra whitespace
-set list listchars=tab:»·,trail:·,nbsp:·
+  let g:go_fmt_command = "goimports"
+  autocmd FileType go compiler go
+  autocmd FileType go setlocal listchars=tab:\ \ ,trail:·,nbsp:·
```

Comment:
I do this slightly differently but don't know enough about the differences between approaches to understand the tradeoffs:

" Don't highlight tabs in Go files
autocmd BufRead,BufNewFile *.go setlocal nolist ft=go